build.sh

```diff
@@ -1011,6 +1017,14 @@ if [ $__CrossBuild == 1 ]; then
 fi
 __CrossGenCoreLibLog="$__LogsDir/CrossgenCoreLib_$__BuildOS.$__BuildArch.$__BuildType.log"
 
+# Configure environment if we are doing a cross compile.
+if [ $__CrossBuild == 1 ]; then
+    export CROSSCOMPILE=1
```

Comment:
What is the env variable `CROSSCOMPILE` being used for?

Reply:
As far as I can tell, it's used to set some platform specific defines

Reply:
This seems fine